[2024-05-04T07:19:23.529+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.write_to_clean_data scheduled__2023-03-04T00:00:00+00:00 [queued]>
[2024-05-04T07:19:23.533+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.write_to_clean_data scheduled__2023-03-04T00:00:00+00:00 [queued]>
[2024-05-04T07:19:23.534+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-05-04T07:19:23.539+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): write_to_clean_data> on 2023-03-04 00:00:00+00:00
[2024-05-04T07:19:23.545+0000] {standard_task_runner.py:57} INFO - Started process 1761 to run task
[2024-05-04T07:19:23.548+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main', 'write_to_clean_data', 'scheduled__2023-03-04T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp2iqqiwj7']
[2024-05-04T07:19:23.551+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask write_to_clean_data
[2024-05-04T07:19:23.585+0000] {task_command.py:415} INFO - Running <TaskInstance: main.write_to_clean_data scheduled__2023-03-04T00:00:00+00:00 [running]> on host cc8c6d1ef843
[2024-05-04T07:19:23.628+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='write_to_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2023-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-04T00:00:00+00:00'
[2024-05-04T07:19:23.693+0000] {warnings.py:109} WARNING - /opt/***/dags/main.py:60: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql_query(query, conn)

[2024-05-04T07:19:32.032+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-05-04T07:19:32.067+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main, task_id=write_to_clean_data, execution_date=20230304T000000, start_date=20240504T071923, end_date=20240504T071932
[2024-05-04T07:19:32.139+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-05-04T07:19:32.161+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
